NMDSdevopsServiceAdm/DataEngineering
# Set validation to error

# Description
Added function to raise a simple error if any data validation checks fail.
Tested new function.

# How to test
Unit tests passing
Run in branch with number of beds minimum value set to 100 (should error out) : https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/set-validation-to-error-validate_locations_api_cleaned_data_job/runs

# Developer checklist
- [x] Linked to Trello ticket : https://trello.com/c/Ay9l7jsy/548-set-up-errors-for-validation
